dbc2val/dbcfeederlib/dbcparser.py

```diff
@@ -28,9 +28,16 @@
 class DBCParser:
     def __init__(self, dbcfile: str, use_strict_parsing: bool = True):
 
-        # Read DBC file
-        log.info("Reading DBC file {}".format(dbcfile))
-        self.db = cantools.database.load_file(dbcfile, strict=use_strict_parsing)
+        self.db = None
+        first = True
+        for filename in dbcfile.split(","):
+            if first:
```

Comment:
here you can strip leading and trailing whitespaces. which lets you specify *.dbc, *.dbc. Also a check if two filenames are the same would be good because else you would get a bunch of warnings with overwriting. Not a priority just a thought.